concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -168,3 +198,26 @@ def test_download_should_return_401_if_wrong_token_cluster_address(self):
         self.assertIsInstance(response, JsonResponse)
         self.assertEqual(response.status_code, 401)
         self.assertIn('message', response.json().keys())
+
+    @freeze_time("2018-12-30 11:00:00")
+    def test_download_should_return_401_if_file_paths_are_not_unique(self):
+        self.upload_token.files.append(FileTransferToken.FileInfo())
```

Comment:
You should assert that `self.upload_token.files` has exactly one element. Otherwise, when someone changes that in `setUp()` your test will just overwrite an existing `FileInfo`.

Also, I think it would be clearer to first create `FileInfo` and only then use `append()`. You would not have to repeatedly refer to its position on the list.

Reply:
I don't understand second part, you want create new `FileInfo` object in method and then create list with two filepaths? It rewrite `FileInfo` object in `setUp()` function.

Reply:
Done.

Reply:
I mean this:

``` python
file1 = FileTransferToken.FileInfo(
    path     = 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend',
    checksum = 'sha1:098f6bcd4621d373cade4e832627b4f6',
    size     = 1024,
)

file2 = FileTransferToken.FileInfo(
    path     = 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend',
    checksum = 'sha1:098f6bcd4621d373cade4e832627b4f6',
    size     = 1024,
)

self.upload_token.files = [file1, file2]
assert file1['name'] == file2['name']
```